### PR TITLE
Pomclean

### PR DESCRIPTION
**Pull request by rodorad**

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -161,12 +156,18 @@
             <artifactId>commons-csv</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.twitter</groupId>
-            <artifactId>parquet-hadoop</artifactId>
+            <groupId>org.parboiled</groupId>
+            <artifactId>parboiled-scala_${scala.short.version}</artifactId>
```
Comment (blbarker) [inline core/pom.xml:160]: I want hard-boiled! !fix

Comment (rodorad) [inline core/pom.xml:160]: !fixed

Comment (rodorad) [inline core/pom.xml:160]: !resolve

Comment (rodorad) [inline core/pom.xml:160]: !resolved

Comment (rodorad) [inline core/pom.xml:160]: fixed

**Pull request merged**